src/test/java/com/algotrading/backtesting/replay/test/ActiveTradedStocksReaderTest.java

```diff
@@ -0,0 +1,58 @@
+package com.algotrading.backtesting.replay.test;
+
+import com.algotrading.backtesting.replay.ActiveTradedStocksReader;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static com.algotrading.constants.TestConstants.*;
+
+public class ActiveTradedStocksReaderTest {
+
+    public static final String VALUE_20130101 = "20130101";
+    public static final String VALUE_20140101 = "20140101";
+
+    private String folderPathStr;
+
+    @Before
+    public void setup() throws IOException {
+        Path folderPath = Files.createTempDirectory("folder");
```

Comment:
This folder will be deleted once the program is over, right?

Reply:
Depends, It is in temp folder (e.g /tmp or C:/temp